src/modules/shared.mjs

```diff
@@ -1,12 +1,13 @@
-function isDesktop() {
+export function isDesktop() {
   return screen.width > 768;
 }
 
```

Comment:
Pebble: `window.innerwidth` rather than `screen.width` to more accurately consider the browser's inner windows width

Reply:
This is a great point!